x-pack/plugins/cloud_security_posture/public/common/hooks/use_url_query.ts

```diff
@@ -35,8 +36,8 @@ export const useUrlQuery = <T extends object>(getDefaultQuery: () => T) => {
     // TODO: condition should be if decoding failed
     if (search) return;
 
-    setUrlQuery(getDefaultQuery());
-  }, [getDefaultQuery, search, setUrlQuery]);
+    replace({ search: encodeQuery(getDefaultQuery() as RisonObject) });
```

Comment:
this is the fix from https://github.com/elastic/security-team/issues/3547 

we need it now since users should be able to go back